### PR TITLE
Accessibility portifolio branch

### PR DESCRIPTION
**Pull request by iankiema**
Updated the following files;
-Index.html

### CODE REVIEW

**Comment by iankiema** [comment]
The following points have been checked;
Page titles - changed the title to Ian Kiema [this link](https://github.com/iankiema/My-Portifolio-Personal/pull/12/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L10-R10)
Image text alternatives - Added alt text to the images [this link](https://github.com/iankiema/My-Portifolio-Personal/pull/12/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L18-R18)
Text headings -  no issues found
Color contrast- no issues found
Resize -  no issues found
Interaction - no issues found
Moving content - no issues found
Multimedia - no issues found
The basic structure of the page - no issues found

File: index.html [hunk 1]
```diff
@@ -7,22 +7,22 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Sofia">
     <link rel="stylesheet" href="style.css">
-    <title>My Personal Portifolio</title>
+    <title>Ian Kiema Portifolio</title>
 </head>
 <body>
```
Comment (bienvenuushindi) [inline index.html:12]: :red_circle: [REQUIRED] I ran a contrast test using  [tool](https://www.aditus.io/button-contrast-checker/)  on your portfolio and found some areas where the color contrast could be improved. Please [follow this link](https://www.aditus.io/button-contrast-checker/cdn-githubraw-com-2023-06-19-at-19-06-12-995?failing) to see the result 

Below is just a screenshot
Firefox_Screenshot_2023-06-19T19-20-04 818Z

**Review by bienvenuushindi — CHANGES_REQUESTED** [review]
Hi Team @iankiema  and @Edgahkipkemoi  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Your project looks good:heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by iankiema** [comment]
The following points have been checked;
Page titles - changed the title to Ian Kiema [this link](https://github.com/iankiema/My-Portifolio-Personal/pull/12/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L10-R10)
Image text alternatives - Added alt text to the images [this link](https://github.com/iankiema/My-Portifolio-Personal/pull/12/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L18-R18)
Text headings - no issues found
Color contrast- fixed the contrast [this link](https://github.com/iankiema/My-Portifolio-Personal/pull/12/files#diff-b78be019f1dc6d57753ea900c3805b114cd53ab7c0db836cc081836df1b99b7aR6-R7)
Resize - no issues found
Interaction - no issues found
Moving content - no issues found
Multimedia - no issues found
The basic structure of the page - no issues found

**Review by divinecharlotte — APPROVED** [review]
# Hello @iankiema       😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say  👍🏻 ✅ 
### To Highlight 💯


- Descriptive pr👍 
- All sections on the page are  👍 
 




## Congratulations! 🎉 



Cheers, and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**